templates/portal/home.html.twig

```diff
@@ -659,9 +659,10 @@
                                     , id: "help-go"
                                 } %}
                                 {{ fireEvent(eventNames.dashboardInjectCard) }}
+                                <!-- Infeg: Updated logout URL to point to portal/logout.php -->
                                 {% include "portal/partial/_nav_icon.html.twig"
                                     with {
-                                    url:  webroot ~ "./logout.php"
```

Comment:
I am pretty sure that replacing `webroot` with `webRoot` would better fix this (now it will break if webRoot is not blank)

Reply:
yep, figured it out after I typed in the post :) Figured i must of been missing something and looked it up (didn't even use AI this time; just a simple google search :) )

Reply:
Yeah, but you still have to be careful with that notation because it can get you in trouble

Reply:
Agreed. I think why we generally try to do complete path. Would of the following have worked (since `webRoot` holds the actual javascript webroot variable in that script):
```
url:  webRoot ~ "/portal/logout.php"
```

Reply:
You’d think. I think I tested and it didn’t work. I think it has to do with how your virtual host is set up.

Reply:
Oh, I see what happened
The bug was Webb root was being combined with the relative path
So you are correct yay, Brady.